modules/ding_place2book/ding_place2book.install

```diff
@@ -107,3 +107,27 @@ function ding_place2book_update_7001() {
     ->fields(array('ticket_type' => 'Adgang'))
     ->execute();
 }
+
+/**
+ * Update ding_event nodes with place2book data.
+ */
+function ding_place2book_update_7002() {
+  $queue = DrupalQueue::get('ding_place2book_update_events_queue');
+  $queue->createQueue();
```

Comment:
The verdict still seems to be out [whether it is necessary to call `createQueue` or not](https://api.drupal.org/api/drupal/modules%21system%21system.queue.inc/function/DrupalQueueInterface%3A%3AcreateQueue/7.x).

If you think it is necessary then it should also be added in `hook_install`.